gdraw/hotkeys.c

```diff
@@ -266,36 +265,36 @@ void hotkeysLoad()
     char* sharedir = getShareDir();
 
     snprintf(localefn,PATH_MAX,"%s/hotkeys/default", sharedir );
-    loadHotkeysFromFile( localefn, false, true );
-    
+    loadHotkeysFromFile( localefn, hk_ff, true );
+
     // FUTURE: perhaps find out how to convert en_AU.UTF-8 that setlocale()
     //   gives to its fallback of en_GB. There are likely to be a bunch of other
     //   languages which are similar but have specific locales
     char* currentlocale = copy(setlocale(LC_MESSAGES, 0));
     snprintf(localefn,PATH_MAX,"%s/hotkeys/%s", sharedir, currentlocale);
-    loadHotkeysFromFile( localefn, false, false );
+    loadHotkeysFromFile( localefn, hk_ff, false );
     while((p = strrchr( currentlocale, '.' ))) {
 	*p = '\0';
 	snprintf(localefn,PATH_MAX,"%s/hotkeys/%s", sharedir, currentlocale);
-	loadHotkeysFromFile( localefn, false, false );
+	loadHotkeysFromFile( localefn, hk_ff, false );
     }
     while((p = strrchr( currentlocale, '_' ))) {
 	*p = '\0';
 	snprintf(localefn,PATH_MAX,"%s/hotkeys/%s", sharedir, currentlocale);
-	loadHotkeysFromFile( localefn, false, false );
+	loadHotkeysFromFile( localefn, hk_ff, false );
     }
     free(currentlocale);
-    
+
     char* fn = getHotkeyFilename(0);
     if( !fn ) {
 	return;
     }
-    loadHotkeysFromFile( fn, true, false );
+    loadHotkeysFromFile( fn, hk_user, false );
     free(fn);
 }
 
 static void hotkeysSaveCallback(Hotkey* hk,FILE* f) {
```

Comment:
I believe that `hotkeysSaveCallback()` was speculative generality anticipating a UI for changing hotkeys. It writes out the entries back to the user's hotkey file. I disabled it in #3606 because automatically writing over a file meant to be hand-edited is annoying.